components/daly_bms_ble/daly_bms_ble.cpp

```diff
@@ -284,7 +286,7 @@ void DalyBmsBle::decode_status_data_(const std::vector<uint8_t> &data) {
   this->publish_state_(this->delta_cell_voltage_sensor_, daly_get_16bit(115) * 0.001f);
 
   // 117   2  0x00 0x2A            Power
-  float power = daly_get_16bit(117) * 1.0f;
+  float power = total_voltage * current;
```

Comment:
It would probably be nice to add a comment explaining why calculation is preferred.

```suggestion
	// Prefer to calculate as the power sensor from the BMS is unsigned. Note the current sensor is signed.
  float power = total_voltage * current;
```